src/components/icon/icon.d.ts

```diff
@@ -1,23 +1,39 @@
 import * as React from 'react';
 
 export interface IconProps {
-  className?: string;
```

Comment:
`className` is still there in the component prop types so should be in here as well still - unless you meant to remove it from the component prop types as well?

Reply:
That prop should be no longer used, I'm leaving it in propTypes to avoid warnings, but I think we're fine to remove it in definition types.

Reply:
Let's add a "DO NOT USE" or something to the prop description then because it's documented in the `Icon` prop table, and nothing says not to use it - so a user could rightly attempt to use it which would give them a Typescript error

Reply:
Yeah sounds good to me 👍🏻